tests/Microsoft.Identity.Test.Performance/Microsoft.Identity.Test.Performance.csproj

```diff
@@ -35,5 +35,6 @@
     <PackageReference Include="MSTest.TestFramework" Version="3.1.1" />
     <PackageReference Update="Microsoft.CodeAnalysis.NetAnalyzers" Version="8.0.0" />
     <PackageReference Update="Microsoft.VisualStudio.Threading.Analyzers" Version="17.8.14" />
+    <PackageReference Update="System.Security.SecureString" Version="4.3.0"/> 
```

Comment:
And why is this needed? We do not use this in the perf project?

Reply:
we do not need this for perf project, this was just a trial and error based on the PR comment from this one. 

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5062